android/source/VideoLocker/src/org/edx/mobile/logger/Logger.java

```diff
@@ -2,10 +2,12 @@
 
 import android.content.Context;
 
+import java.io.Serializable;
+
 /**
  * Created by shahid on 22/1/15.
  */
-public class Logger {
+public class Logger implements Serializable {
```

Comment:
This is required when using <code>Logger</code> inside activities or fragments when screen orientation change is allowed. I am not sure if this is the best way, but otherwise, we need to use static instance of logger which we don't want.

Making it <code>Serializable</code> allows logger instance to be saved and restored across activity re-creations.